packages/vats/src/lib-chainStorage.js

```diff
@@ -53,27 +53,27 @@ export function makeChainStorageRoot(
 
   function makeChainStorageNode(path) {
     const node = {
+      /** @type {() => VStorageKey} */
       getStoreKey() {
         return handleStorageMessage({ key: path, method: 'getStoreKey' });
       },
-      getChildNode(name) {
+      /** @type {(name: string) => VStorageNode} */
+      makeChildNode(name) {
```

Comment:
In Agoric conventions that I learned in the past few weeks, I think the most appropriate name for this method is `provideChildNode` ("provide" referring to operations that return a value after creating it if necessary, acknowledging that it might have already existed).

And likewise for `makeStorageNodeChild`.

Reply:
I think with `provide` you get the same object on repeated requests. The two methods you suggest renaming actually call `makeChainStorageNode` which always makes a fresh node. Changing that is out of scope but I can follow up to do that. WDYT @michaelfig ?